hi-ml/src/health_ml/networks/layers/attention_layers.py

```diff
@@ -24,6 +24,21 @@ def forward(self, features: Tensor) -> Tuple[Tensor, Tensor]:
         return (attention_weights, pooled_features)
 
 
+class MaxPoolingLayer(nn.Module):
+    """Max pooling returns uniform weights and the maximum feature vector over the first axis"""
+
+    # args/kwargs added here for compatibility with parametrised pooling modules
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
This empty constructor is no longer necessary after Max's PR https://github.com/microsoft/hi-ml/pull/227.

Reply:
I have removed this now, thanks.